doc/rpc/gettxout.md

```diff
@@ -0,0 +1,47 @@
+# `gettxout`
+
+Returns details about an unspent transaction output.
+
+## Arguments
+
+* `txid` - (string, required) The transaction id;
+* `command` - (string, required) vout number;
+* `include_mempool` - (not implemented).
+
+## Returns
+
+### Ok Response
+
+* json
+
+```json
+{  
+  "bestblock" : "hex",      (String) The hash of the block at the tip of the chain
+  "confirmations" : n,    (Numeric) The number of confirmations
+  "value" : n,            (Numeric) The transaction value in BTC
+  "scriptPubKey" : {
+    "asm" : "hex",           (String)
+    "hex" : "hex",           (String)
+    "type" : "hex",          (String) The type, eg pubkeyhash
+    "addresses" : "str"      (String) bitcoin addresses
+  },
+  "coinbase" : true|false  (Boolean) Coinbase or not
+}
+```
+
+### Error Enum `CommandError`
+
+Any of the error types on `rpc_types::Error`
```

Comment:
I think this can be more expressive about which errors you should reasonably expect (we didn't find the output, our node died....)